### DISCUSSION TITLE
Progress Report 25 – 29 April 2022

### DISCUSSION THREAD

**Discussion by esheehan-gsl** [NOAA-GSL/unified-graphics#13] [Progress Report]
🚴🏻 **Progress**
- Discussed data assimilation diagnostics with @terraladwig, @guoqing-noaa, @christinaholtNOAA, and @ian-noaa 
- Began planning scope of work for first demo: #11
- Discussed AWS infrastructure with VLAB team

😖 **Regressions**
- None

💬 **To Discuss**
- Nothing

📆 **Next Week**
- Scaffold the code for the frontend including [USWDS](https://designsystem.digital.gov/) for the styles

**Comment by ian-noaa** [comment]
Thanks, Evan. One addition.

 :calendar: Next Week
* Make a formal request for resources in VLab's cloud
* Set up AWS managed Kubernetes (EKS) in VLab's cloud
